src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -50,13 +53,16 @@ public ActivityAttribute ()
 #if ANDROID_26
 		public float                  MaxAspectRatio          {get; set;}
 #endif
+		public int                    MaxRecents              {get; set;}
 		public bool                   MultiProcess            {get; set;}
 		public bool                   NoHistory               {get; set;}
 #if ANDROID_16
 		public Type                   ParentActivity          {get; set;}
 #endif
+		public ActivityPersistableMode      PersistableMode   {get; set;}
```

Comment:
This was added in API-21, and thus the declaration should be wrapped in a `#if ANDROID_21` block.